bciers/apps/administration/app/components/profile/ProfileForm.tsx

```diff
@@ -55,6 +48,75 @@ export default function ProfileForm({ formData, isCreate }: Props) {
   // ✅ Success state for for the Submit button
   const [isSuccess, setIsSuccess] = useState(false);
 
+  const [emailHelpText, setEmailHelpText] = useState<JSX.Element | null>(null);
+
+  const emailHelpTextFirstClause = (
+    <div>
+      This email is used to log in. To change your login email, contact{" "}
+      <a href="mailto:ghgregulator@gov.bc.ca">ghgregulator@gov.bc.ca</a>
+    </div>
+  );
+
+  useEffect(() => {
+    const fetchHelpText = async () => {
+      // Create help text for email address field depending on several conditions:
+      // 1. if the user is internal, no help text is required
+      // 2. if it's an external user and their profile is just being created, there is no contact for them and they shouldn't be able
+      // to navigate away from the Profile page
+      // 3. if it's an external user who already has a contact record for themselves, then they should also see the link to their contact details page
+      const session = await getSession();
+      const idp = session?.identity_provider || "";
+      const businessGuid = session?.user?.bceid_business_guid;
+
+      // if the user is internal, no help text is required
+      if (idp === "idir") {
+        setEmailHelpText(null);
+        return;
+      }
+
+      // if the user is external but being created, there's no contact record for them, so
+      // the help text message is simplified
+      if (isCreate) {
+        setEmailHelpText(emailHelpTextFirstClause);
+        return;
+      }
+
+      // if the user is external and a contact record already exists for them, the
+      // help text message is more detailed
+      if (businessGuid) {
+        try {
+          const response = await actionHandler(
+            `registration/contact/`,
+            "GET",
+            "/profile",
+          );
+
+          if (response) {
+            setEmailHelpText(
+              <>
+                {emailHelpTextFirstClause}
+                <div>
+                  To change the email you are contacted with, edit the email in
+                  your{" "}
+                  <a href={`administration/contacts/${response}`}>
+                    contact details page
+                  </a>
+                  .
+                </div>
+              </>,
+            );
+          } else {
+            // fallback if no contact found
+            setEmailHelpText(emailHelpTextFirstClause);
+          }
+        } catch (e) {
+          setEmailHelpText(emailHelpTextFirstClause);
+        }
+      }
+    };
+    fetchHelpText();
+  }, [isCreate]);
```

Comment:
I think we could avoid this whole 'useEffect', 'getSession' and logic by running this in a server side component. We can then pass the contact id as a parameter for the client side page to render the help text or not.
The server side component might also be able to make use of the `getUserRole` utility method that already exists